processor/src/main/java/org/mapstruct/ap/internal/model/StreamMappingMethod.java

```diff
@@ -78,14 +78,16 @@ protected StreamMappingMethod instantiateMappingMethod(Method method, Collection
             );
         }
     }
-
-    private StreamMappingMethod(Method method, Collection<String> existingVariables, Assignment parameterAssignment,
+    //CHECKSTYLE:OFF
```

Comment:
When you turn of Checkstyle you should turn it back on with `//CHECKSTYLE:ON`. Have a look at how it is done in the `BeanMappingMethod` constructor and please do it like that